src/test/ts/browser/InitTest.ts

```diff
@@ -47,7 +47,7 @@ UnitTest.asynctest('InitTest', (success, failure) => {
             <editor
               :init="init"
               v-on:input="content = $event"
-              outputFormat="text"
+              output-format="text"
```

Comment:
I'm guessing this wasn't working properly before? If so then, maybe the test isn't actually working properly since it didn't catch this, or did something else change?

Reply:
Both works and Vue does a conversion for the props but kebab-case is the recommended convention in templates

Reply:
Ahh didn't know that, thanks @SimonFc :+1: